nemo_skills/inference/eval/hilbench_assets/ask_human_server.py

```
@@ -0,0 +1,835 @@
+                    latest_entry["blocker_name"] = None
+                    response = IRRELEVANT_QUESTION
+
+            GLOBAL_LOGS[instance_id]["questions"].extend(tool.log["questions"])
+            # Update blockers status (merge True values)
+            for blocker_key, resolved in tool.log["blockers"].items():
+                if resolved:
+                    GLOBAL_LOGS[instance_id]["blockers"][blocker_key] = True
+
+        return jsonify({"response": response})
+
+    except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)